### ISSUE TITLE
[Accessibility team] Show/Hide Columns button controls are not accessible using keyboard in A11yCluster tab

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#309]
[ADO LINK](https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/36820327)

prio 2
sev 3

### Actual: 
In the A11yCluster tab, the Show/Hide Columns button controls are not operable using keyboard navigation.

Note: This issue is observed throughout the application for all Show/Hide Columns button.

### Expected:
Show/Hide Columns button controls should be reachable via keyboard.

### User Impact:
Keyboard-only users and screen reader users cannot customize visible columns, limiting access to data and functionality.

### Repro Steps:

- Launch AKS desktop application.
- TAB to "Azure Account" button and press ENTER.
- Azure Authentication login screen will open. Sign in with your v-id.
- Afte login "Azure Account" dialog will open with "Tenant ID" and "Default subscription ID". TAB to "Add cluster from Azure" button and press enter.
- "Register AKS Cluster" dialog will open. Verify all the elements of these dialogs are accessible and MAS compliant.
- After adding the cluster home page will open.
- TAB to "All Clusters" Tab and press ENTER. 
- TAB to "Show/Hide Columns" button and press ENTER.
- Observe that Show/Hide Columns button controls are not accessible using keyboard in A11yCluster tab.
